docs/advanced-customization/config/emails.md

```diff
@@ -1,10 +1,21 @@
 ---
-description: Set your community email address.
-position: 5
+description: Set your community email address and periodic email digest.
```

Comment:
We've reworked the Emails section to remove the default email which was disabled anyways. We've added a contact email and moved the email digest field to the emails section